modules/electromagnetics/test/tests/auxkernels/azimuthal_Faradays_law/scalar_azim_magnetic_time_deriv.i

```diff
@@ -0,0 +1,243 @@
+# Test for AzimuthMagneticTimeDerivScalarRZ
+# Manufactured solution: E_real = y^2 * x_hat - x^2 * y_hat
+#                        E_imag = y^2 * x_hat - x^2 * y_hat
+#                        dB_theta_real / dt = -(2*y + 2*x)
+
+[Mesh]
+  [gmg]
+    type = GeneratedMeshGenerator
+    dim = 2
+    nx = 10
+    ny = 10
+    xmax = 1
+    xmin = 0
+    ymax = 1
+    ymin = -1
+    elem_type = QUAD9
+  []
+  coord_type = RZ
+  rz_coord_axis = Y
+[]
+
+[Functions]
+  #The exact solution for the heated species and E-field real and imag. component
```

Comment:
Let's use "electric field" in input file comments too.

Reply:
Edited in [70a7473](https://github.com/idaholab/moose/pull/28759/commits/70a7473b7affe5e6fce2a562654a3f55bc306c9f)